www/resources/campus-idp.css

```
@@ -67,6 +67,18 @@ html, body {
     margin-bottom: 24px;
 }
 
+.framework_muni .remembered-institution {
+    margin-top: 12px !important;
+}
+
+.remembered-institution {
+    margin-top: 12px;
+}
```

```suggestion
.remembered-institution {
    margin-top: 12px !important;
}
```
`!important` can be left for Bootstrap, too (it does nothing when there are no conflicts)